packages/manager/cypress/e2e/core/cloudpulse/timerange-verification.spec.ts

```diff
@@ -128,15 +128,14 @@ const mockProfile = profileFactory.build({
  *   - `month`: The month of the year as a number.
  */
 const getDateRangeInGMT = (
-  daysOffset: number,
   hour: number,
-  minute: number = 0
+  minute: number = 0,
+  isCurrent: boolean = false
 ) => {
   const now = DateTime.now().setZone('GMT'); // Set the timezone to GMT
-  const targetDate = now
-    .startOf('month')
-    .plus({ days: daysOffset })
-    .set({ hour, minute });
+  const targetDate = !isCurrent
+    ? now.set({ hour, minute })
+    : now.startOf('month').set({ hour, minute });
```

Comment:
```suggestion
  const targetDate = isCurrent
    ? now.startOf('month').set({ hour, minute }): now.set({ hour, minute });
```

This looks more readable

Reply:
I have addressed  the comments